src/Microsoft.CodeAnalysis.Analyzers/Core/MetaAnalyzers/Fixers/PreferIsKindFix.cs

```diff
@@ -48,7 +47,7 @@ private async Task<Document> ConvertKindToIsKindAsync(Document document, TextSpa
 
         protected abstract void FixDiagnostic(DocumentEditor editor, SyntaxNode nodeToFix);
 
-        private sealed class CustomFixAllProvider : DocumentBasedFixAllProvider
+        private sealed class CustomFixAllProvider : Analyzer.Utilities.DocumentBasedFixAllProvider
```

Comment:
📝 Note to self: review this unexpected change

Reply:
📝 It appears this library is building against the wrong set of reference assemblies. This should be corrected, at which point no changes are required to this file.

Reply:
See https://github.com/dotnet/installer/blob/main/src/SourceBuild/patches/roslyn-analyzers/0001-Eliminate-pre-built-assets-during-source-build-for-r.patch for places where this change was made. I believe there are conflicting type names?